docs/reference/metadata.md

```diff
@@ -911,7 +911,7 @@ The ``funder`` subfields:
 | ``id`` | (0-1, CV) | The funder id from the controlled vocabulary. |
 | ``name`` | (0-1) | The name of the funder. |
 
-One of ``id`` or ``name`` must be given. It's recommended to use ``name`` if there is no matching ``id`` in the controlled vocabulary.
+Either ``id`` or ``name`` must be given. It's recommended to use ``name`` if there is no matching ``id`` in the controlled vocabulary.
```

Comment:
I am actually not in favor of using "Either". It is a very ambiguous word. Just taking the https://wordnik.com/words/either entry: it can mean:

- exclusive or (as in: one or the other but NOT both)
OR
- and (as in: one AND the other)

The previous formulation ("one of") was reflecting the code which really wanted one OR the other. 

Maybe using all caps for OR would be clearer?

Reply:
Interesting, I've never heard of that use of either before. I've taken your suggestion and revised.